### PR TITLE
1895 Batch entity

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: shared/src/business/entities/Batch.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * @constructor
+ */
+function Batch(rawBatch) {
+  this.batchId = rawBatch.batchId || uuid.v4();
+  this.batchIndex = rawBatch.batchIndex || 0;
+  this.createdAt = rawBatch.createdAt || new Date().toISOString();
+  this.pages = rawBatch.pages || [];
+}
+
+Batch.name = 'Batch';
```
Comment (kkoskelin) [inline shared/src/business/entities/Batch.js]: If this is necessary, please use `Batch.validationName`.  I think it causes problems with some new linting rules.

**Review by kkoskelin — APPROVED** [review]

**Review by ericsorenson — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

File: shared/src/business/entities/Batch.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+const sinon = require('sinon');
```
Comment (kkoskelin) [inline shared/src/business/entities/Batch.test.js]: You'll have to remove `sinon` for tests to pass (unused var)

Comment (sutt0n) [reply shared/src/business/entities/Batch.test.js]: switched to vscode, apparently the editor didn't catch it when i was modifying the file at the time 😢

**Pull request merged**